superset/utils/webdriver.py

```diff
@@ -219,15 +217,16 @@ def get_screenshot(
             img = element.screenshot_as_png
 
         except TimeoutException:
-            logger.warning("Selenium timed out requesting url %s", url, exc_info=True)
+            logger.exception("Selenium timed out requesting url %s", url)
```

Comment:
This is going to give us a lot of errors in the application that aren't necessarily system errors, but could be a result of a user error or an issue with a user's database performance. Are we sure we want to log these as exceptions? I'm currently seeing the stack trace under exc_info in the logs?

Reply:
I see what you're saying, but it's my opinion that exceptions should be logged as such. If there's timeouts, we should adjust the timeout value in order to prevent this. If there's some network issue, etc. we should look into that as well.

Reply:
That makes sense. I think some of these could be user-related, as in the user's dataset is broken. Maybe we can split out the errors to get a better sense of what's going on, and separate out the system errors from the user errors. I put together a PR here.. lmk what you think. https://github.com/apache/superset/pull/23290